src/routes/demographics/Content.svelte

```diff
@@ -175,6 +176,29 @@
     },
   };
 
+  function questionIsAnswered(answer, questionType) {
+      if (questionType === 'text') return answer.length > 0;
+      if (questionType === 'single') return answer !== undefined;
+      if (questionType === 'multi') return answer.length > 0;
```

Comment:
Should this also throw an error like `clearAnswer()` does if an unknown `questionType` is passed?

Also - this is just a nit you can take or leave it - this feels repetitive but if we're not going to add any further question types in the future it doesn't really matter. I'm surprised our eslint rules allow no-braces as well.

Writing it like this would allow you to also have an `else` branch to throw an error or log:
```
  if (questionType === "text" || questionType === "multi") {
    return answer.length > 0;
  } else if (questionType === "single") {
    return answer !== undefined;
  } else { ... }
```
It doesn't line up as nicely visually as yours though :)